docs/user-guide/services/index.md

```diff
@@ -193,7 +193,10 @@ default is `"None"`).
 As with the userspace proxy, the net result is that any traffic bound for the
 `Service`'s IP:Port is proxied to an appropriate backend without the clients
 knowing anything about Kubernetes or `Services` or `Pods`. This should be
-faster and more reliable than the userspace proxy.
+faster and more reliable than the userspace proxy. However, unlike the
+userspace proxier, the iptables proxier cannot automatically retry another
+`Pod` if the one it initially selects does not respond, so it depends on
+having working [readiness probes](https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/user-guide/production-pods.md#liveness-and-readiness-probes-aka-health-checks).
```

Comment:
Change URL to /docs/user-guide/production-pods/#liveness-and-readiness-probes-aka-health-checks